Filtered_kernel/include/CGAL/Lazy.h

```diff
@@ -781,7 +781,7 @@ public :
     ptr()->print_dag(os, level);
   }
 
-private:
+  //private:
```

Comment:
I think the patch need cleaning, now that it works. Could not the Lazy_kernel be friend of this class, instead of having to turn it into a `struct`?